app/models/order.rb

```diff
@@ -30,12 +30,12 @@ class Order < ApplicationRecord
 
   # Finders
   scope :started, -> { where('starts <= ?', Time.now) }
-  scope :open, -> { where(state: 'open').order('ends DESC') }
-  scope :finished, -> { where("orders.state = 'finished' OR orders.state = 'closed'").order('ends DESC') }
-  scope :finished_not_closed, -> { where(state: 'finished').order('ends DESC') }
-  scope :closed, -> { where(state: 'closed').order('ends DESC') }
-  scope :stockit, -> { where(supplier_id: nil).order('ends DESC') }
-  scope :recent, -> { order('starts DESC').limit(10) }
+  scope :open, -> { where(state: 'open').order('orders.ends DESC') }
+  scope :finished, -> { where("orders.state = 'finished' OR orders.state = 'closed'").order('orders.ends DESC') }
+  scope :finished_not_closed, -> { where(state: 'finished').order('orders.ends DESC') }
+  scope :closed, -> { where(state: 'closed').order('orders.ends DESC') }
+  scope :stockit, -> { where(supplier_id: nil).order('orders.ends DESC') }
+  scope :recent, -> { order('orders.starts DESC').limit(10) }
```

Comment:
I got an error in one of the queries when doing this (because `starts` can be ambiguous when joining with a model having the same attribute). In a rebase I've replaced this with `order(starts: :desc)` so that Rails does this automatically.

Reply:
thx, `order(starts: :desc)` is what i expected